buildlib/azure-pipelines-release.yml

```diff
@@ -42,6 +42,15 @@ stages:
                 ./ucx-*.tar.gz
                 ./rpm-dist/ucx-*.src.rpm
 
+          - bash: ./autogen.sh
+            displayName: Setup autotools
+
+          - bash: |
+              set -eE
+              ./contrib/configure-release --with-java
```

Comment:
line 28 is already doing with-java, why need again here?

Reply:
`rpmbuild.sh` cleans everything after it's finished. That's why there's no `libjucx.so` in the jar. `--with-java` to make sure it configured with java,  before publishing step. There could be other steps before, some may reconfigure, etc.

Reply:
you mean `./contrib/buildrpm.sh` scrip? it shouldn't clean.. where do you see it?